services/ui/ws/window_tree.cc

```diff
@@ -900,6 +900,13 @@ void WindowTree::SendToPointerWatcher(const ui::Event& event,
                                    display_id);
 }
 
+WindowManagerState* WindowTree::AppendExternalModeWindowManagerState(
```

Comment:
even by doing this, I still see on console:

[19255:19255:0402/234231.834795:15837186272:ERROR:mash_runner.cc(149)] Main process exiting because service content_packaged_services quit unexpectedly.

@fred-wang: do you think it is ok to merge it?